lib/p5.speech.js

```diff
@@ -358,14 +355,33 @@
   // this one 'start' cycle.  if you need to recognize speech more
   // than once, use continuous mode rather than firing start()
   // multiple times in a single script.
-  p5.SpeechRec.prototype.start = function() {
+  p5.SpeechRec.prototype.start = function(_continuous, _interimResults) {
```

Comment:
@AlcaDesign Can you take a look at #8 which also addresses this function, but in a slightly different way?

Reply:
The nice thing about doing it in @AlcaDesign's way is that it works with both property modification and arguments to start. That's not how I would design the API myself but there's value in keeping the API close to webkitSpeechRecognition so supporting property modification is a good idea. I suggest splitting the line up into two parts to increase clarity: first set the properties based on the arguments using if-statements, and then use the properties.

Reply:
Agreed, go ahead and close your pull request if you like @Zalastax! Thanks for your contribution!